.github/workflows/new-migrated-issue.yml

```diff
@@ -0,0 +1,44 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      name:
+        description: name
+        required: true
+        type: string
+
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  receive:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest 
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      - uses: nickderobertis/check-if-issue-exists-action@master
+        name: Check if Issue Exists
+        id: check_if_issue_exists
+        with:
+          repo: noir-lang/docs
+          token: ${{ secrets.GITHUB_TOKEN }}
+          title: Document noir PR ${{ inputs.name }}
+          labels: migrated
+          
+      # Runs a single command using the runners shell
+      - name: create an issue
+        if: steps.check_if_issue_exists.outputs.exists == 'false'
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ github.token }}
+          title: Document noir PR ${{ inputs.name }}
+          labels: migrated
+          body: Document noir PR [${{ inputs.name }}](https://github.com/noir-lang/docs/pull/${{ inputs.name }}) 
```

Comment:
Nitpick but it would be nice to standardise on a single action for creating new issues. We have one in the `build-nargo` repo which allows you to use some nicer templates and assign documentation writers, etc.

https://github.com/noir-lang/build-nargo/blob/bc0008a2473c052ffc6a6690f388898fd196ca8c/.github/workflows/publish-linux.yml#L96-L104